surfsense_backend/app/utils/connector_service.py

```
@@ -373,65 +425,75 @@ async def search_extension(self, user_query: str, user_id: str, search_space_id:
-                        duration_text = f"{duration_seconds:.1f} seconds"
-                    else:
-                        duration_text = f"{duration_seconds/60:.1f} minutes"
+                # Create a more descriptive title for extension data
+                title = webpage_title
+                if visit_date:
+                    # Format the date for display (simplified)
+                    try:
+                        # Just extract the date part for display
+                        formatted_date = visit_date.split('T')[0] if 'T' in visit_date else visit_date
+                        title += f" (visited: {formatted_date})"
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)